frontend/src/main/utils/usersUtils.js

```diff
@@ -0,0 +1,26 @@
+import { toast } from "react-toastify";
+
+export function onDeleteSuccess(message) {
+    console.log(message);
+    toast(message);
+}
+
+export function cellToAxiosParamsDelete(cell) {
```

Comment:
See, it's not a delete!  It's a PUT to the `hide` endpoint.  So it's very confusing to call it delete.